keepalived-vip/keepalived.go

```diff
@@ -68,54 +68,73 @@ vrrp_instance vips {
   }
 }
 
-{{ range $i, $svc := .svcs }}
```

Comment:
There already is a check for .routes from line 32 to 38. 
It currently always generates both routes and rules if they are available. We could add a switch for rules though. I did not add that switch for because the template can now changed by the user.